frontend/storybook/storybook/stories/Icon/Icon.stories.js

```diff
@@ -1,21 +1,20 @@
 import { storiesOf } from '@storybook/react-native';
 import React from 'react';
-import { Icon } from 'core-design-system';
-import { IconName, IconVariants } from 'core-design-system/lib/esm/types/icons';
+import { Icon, icons, theme, tokens } from 'core-design-system';
```

Comment:
If you don't want to have `icons` as an export of the package, you could do:
```
import icons from 'core-design-system/lib/esm/cant-remember-the-path';
```

But I think the export is fine